src/MensaWidget/TheWorldMainDockingBar.extension.st

```diff
@@ -0,0 +1,98 @@
+Extension { #name : #TheWorldMainDockingBar }
+
+{
+	#category : #'*MensaWidget-override',
+	#'squeak_changestamp' : 'Louis Christ 6/9/2024 15:10'
+}
+TheWorldMainDockingBar >> mensaMenuOn: aDockingBar [
+	"Create a menu with the options for MensaWidget"
```

Comment:
Could we rename the method so we don’t need the comment? Something like ‘createOptionsMenuMW’?

Reply:
Ich stimme dir zu, dass man den Kommentar redundant machen könnte. Allerdings orientiert sich der Stil des Namens an den anderen wie beispielsweise "helpMenuOn" oder "windowsMenuOn". Da alle Methodennamen in dem Stil sind, würde ich mich vielleicht auch erstmal daran orientieren, da wir hier schon ziemlich an unserer Domänengrenze sind

Reply:
I agree with the fact that the comment could be made redundant. On the other hand, all of the method names follow the same style, e.g. "helpMenuOn" or "windowsMenuOn". Since we are borderline leaving our domain here I'd stick to the existing scheme